### PR TITLE
December release notes

### PR DESCRIPTION
**Pull request by jcrossley**

### CODE REVIEW

File: source/blog/2016-12-23-release-notes-6-41.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+---
+
+As the year ends, we've got an exciting new release for you! Here's a quick rundown
+of what we've been up to.
+
+### Scala 2.12 Support is Here
+
+Finagle, Util, Ostrich, Scrooge, and Twitter-Server are all now cross-compiled for Scala 2.11 and 2.12(!)
+
+Dynamic Timouts
```
Comment (taylorleese) [inline source/blog/2016-12-23-release-notes-6-41.md]: Was this suppose to be its own section with ###?

File: source/blog/2016-12-23-release-notes-6-41.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+The new [Dynamic Timeoute module][dynamic timeouts] module allows clients to specify call-site specific timeouts.
```
Comment (taylorleese) [inline source/blog/2016-12-23-release-notes-6-41.md]: Timeoute => Timeout

File: source/blog/2016-12-23-release-notes-6-41.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+The new [Dynamic Timeoute module][dynamic timeouts] module allows clients to specify call-site specific timeouts. 
+
+### Deadline Filter Rises from the Dead
+
+We've resurrected [Deadline Filter][deadline filter] as an optional module that can be added to clients
+and servers to reject requests with expired deadlines. If your calling clients are
+well-prepared to handle the resulting NACKs, check out this filter.
+
+### Filtered ThriftMux Clients
+You can now added filters (using the `.filtered` method) on ThriftMux stack clients.
```
Comment (taylorleese) [inline source/blog/2016-12-23-release-notes-6-41.md]: added => add

File: source/blog/2016-12-23-release-notes-6-41.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+### Turn it up to a 1.0
+Toggles prove a convenient way to slowly enable functionality. We've now promoted them
```
Comment (taylorleese) [inline source/blog/2016-12-23-release-notes-6-41.md]: ffti enable => rollout new

**Comment by jcrossley** [comment]
Updated per Taylor's feedback, thanks!

File: source/blog/2016-12-23-release-notes-6-41.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+---
+
+As the year ends, we've got an exciting new release for you! Here's a quick rundown
+of what we've been up to.
+
+### Scala 2.12 Support is Here
+
+Finagle, Util, Ostrich, Scrooge, and Twitter-Server are all now cross-compiled for Scala 2.11 and 2.12(!)
+
+### Dynamic Timouts
```
Comment (taylorleese) [inline source/blog/2016-12-23-release-notes-6-41.md]: Timouts => Timeouts (sorry missed this one before)

**Comment by jcrossley** [comment]
correct spelling

**Comment by vkostyukov** [comment]
LGTM! @jcrossley Please squash your commits into one to keep the git history clean.

**Pull request merged**